pandas/tests/util/test_show_versions.py

```diff
@@ -1,8 +1,26 @@
 import re
 
+import pytest
+
 import pandas as pd
 
 
+@pytest.mark.filterwarnings(
```

Comment:
Are these actually required as part of this change? The linkage here with the rest of the PR isn’t totally clear

Reply:
these are just because its annoying to have in the output